x-pack/plugins/security_solution/public/detection_engine/rule_creation/components/related_integrations/related_integrations_help_info.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import { EuiLink, EuiPopover, EuiText, EuiButtonIcon } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '../../../../common/lib/kibana';
+
+const POPOVER_WIDTH = 320;
```

Comment:
Is this the conventional popover width? Should this be referenced through a shared constant or EUI theme variable, rather than a magic number here? Did design designate this width, or: where did the number come from?

Reply:
It's a good point. I tried to find width theme variables but EUI doesn't expose width variables. `320` is the hardcoded value [used in different components in EUI](https://github.com/search?q=repo%3Aelastic%2Feui%20320&type=code) . It's [also used](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/suppression_info_icon/index.tsx#L14) for suppression info popover on rule create page. I'll add a convey this information.